docs/convert.md

```diff
@@ -0,0 +1,326 @@
+# Eagle Checkpoint Conversion Guide
+
+This guide explains how to convert EAGLE 1, EAGLE 2, and HASS checkpoints to the standardized speculators format.
+
+## Overview
+
+The speculators library provides a unified interface for speculative decoding models. To use existing Eagle/HASS checkpoints, they must first be converted to the speculators format.
+
+## Supported Checkpoints
+
+We support converting the following checkpoint types:
+
+- **EAGLE 1**: Original Eagle architecture
+- **EAGLE 2**: Updated Eagle architecture (same structure as EAGLE 1)
+- **HASS**: Hardware-Aware Speculative Sampling variant
+
+## Quick Start
+
+```bash
+# Install speculators
+pip install speculators
+
+# Convert a standard Eagle checkpoint
+speculators convert eagle yuhuili/EAGLE-LLaMA3.1-Instruct-8B ./converted/eagle meta-llama/Llama-3.1-8B-Instruct
```

Comment:
remove "eagle" from subcommand structure, add in --argument or something along those lines